### DISCUSSION TITLE
Which is the best version for ES 7.6

### DISCUSSION THREAD

**Discussion by larryzk** [alexklibisz/elastiknn#745] [Q&A] [answered]
I have to use version 7.6 of ES. Which version is the best in terms of stability and compatibility?

**Comment by alexklibisz** [comment, marked as answer]
Hi, there is no release for 7.6. 7.6 came out in February 2020. There are some 7.x releases. You can look at the tags to find them. 

<img width="446" alt="image" src="https://github.com/user-attachments/assets/c627f3ec-234f-42d8-89ac-07386232662f">

I recommend questioning why you're using (or being told to use) a version thats almost 5 years old.
